example/cumulus-tf/variables.tf

```diff
@@ -338,7 +338,7 @@ variable "rds_admin_access_secret_arn" {
 variable "async_operation_image_version" {
   description = "docker image version to use for Cumulus async operations tasks"
   type = string
-  default = "36"
+  default = "39"
```

Comment:
As noted in the comments, this needs to be pushed to ECR for our tests to work, and if we want users to use 39, we'll need to push it to cumuluss and update the default in the`cumulus` module.